Marlin/stepper.cpp

```diff
@@ -593,7 +595,7 @@ void Stepper::isr() {
     #endif // ADVANCE or LIN_ADVANCE
 
     #if ENABLED(ADVANCE) || ENABLED(LIN_ADVANCE)
-      eISR_Rate = (timer >> 2) * step_loops / abs(e_steps[TOOL_E_INDEX]);
+      eISR_Rate = (timer >> 3) * step_loops / abs(e_steps[TOOL_E_INDEX]); //>> 3 is divide by 8. Reason: Timer 0 runs at 16/8=2MHz, Timer 1 at 16/64=0.25MHz. ==> 2/0.25=8.
```

Comment:
Is "Timer 1 runs at 16/8=2MHz, Timer 0 at 16/64=0.25MHz" right?

Because,
`Stepper::isr()` ... `TIMER1_COMPA_vect`
`Stepper::advance_isr()` ... `TIMER0_COMPA_vect`